app/Model/Category.php

```diff
@@ -39,4 +39,12 @@ protected static function mapOne( $data)
     {
 
     }
+
+    protected static function getTable(): string
+    {
+        if (static::$table !== null) {
+            return static::$table;
+        }
+        return "Table not defined!";
```

Comment:
Returning a string message is fault,
You probably need to return table name directly 
return 'category'; or a cost self::TABLE_NAME